src/equations/acoustic_perturbation_2d.jl

```diff
@@ -0,0 +1,153 @@
+@doc raw"""
+    AcousticPerturbationEquations2D(v_mean, rho_mean, c_mean)
+
+Acoustic perturbation equations in two space dimensions with constant mean flow. The equations are
+based on the APE-4 system introduced by Ewert and Schröder
```

Comment:
Could you please extend this citation with some more basic information on the paper (title, journal, year)?